core/core.md

```diff
@@ -62,14 +62,16 @@ Thus the Fora protocol must assume that malicious actors will never reach the th
 
-This gives the community enough time to detect the attack and replace the Admittor before the Admittor takes over community governance, while also allowing new members to immediately participate in the community and giving them voting power later on.
+This gives the community enough time to detect the attack and replace the Admittor before the Admittor takes over community governance, while also allowing new members to immediately participate in the community and giving them voting power later on. Admittors who try to admit malicious members over a longer period of time are also constrained by their own voting power obtained through a governace proposal.
 
-If the Admittor gets replaced for malicious activity, then any member that is still in the delay period will have their voting power remain 0. The new Admittor must **explicitly** choose to give back the new members their voting power, at which point the delay period gets reset and must pass again from that point. However, these members can still interact with the community unless the admittor or enforcer **explicitly** kicks them out of the community.
+If the Admittor gets replaced for malicious activity, then any new member that is still in the delay period will have their voting power remain 0. The voting power given to users from this admittor will also be removed. The new Admittor must **explicitly** choose to give back the new members their voting power, at which point the delay period gets reset and must pass again from that point. However, these members can still interact with the community unless the admittor or enforcer **explicitly** kicks them out of the community.
 
 Admittors may remove members who are still in the delay period, but cannot remove established members. This gives established members protection from admittors but allows admittors to kick out new members who were maliciously added by an old admittor or to simply cull inactive new members.
 
+Multiple admittors can grant voting powers to a single user. This way honest users can obtain more voting power in the community as the user is recognized by multiple admittors. When an admittor is kicked or replaced due to a governance proposal, only the weight attributed to this removed admittor is substracted by the users total voting power.
```

Comment:
Again, established members shouldn't automatically have their voting power revoked because of a malicious admittor.

Reply:
Yea the benefit here should be when an admittor tries reducing a members voting power ie kicking them, if they are vouched for by multiple admittors, the other admittors would also need to kick them